### ISSUE TITLE
outdated wiki information

### ISSUE THREAD

**Issue by JuliaSprenger** [G-Node/nix#831]
In the wiki, e.g. https://github.com/G-Node/nix/wiki/Model-Definition, an old version of the odML library is depicted, that still features `value` objects.

**Comment by jgrewe** [comment]
Hi @JuliaSprenger actually, it is still valid. In fact, the 1.4.x versions in nix still use the old style odml model. This will change with the 1.5 release and then this info indeed needs to be update. 
I will keep this issue open as a reminder to make the changes once 1.5 is released

**Comment by JuliaSprenger** [comment]
Yes, you are right. I forgot that the nixio 1.5 is still in beta mode.
